packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -883,20 +883,34 @@ export default class SalesforceCRMService implements CRM {
       });
   }
 
+  private getAllPossibleAccountWebsiteFromEmailDomain(emailDomain: string) {
+    return [
+      emailDomain,
+      `www.${emailDomain}`,
+      `http://www.${emailDomain}`,
+      `http://${emailDomain}`,
+      `https://www.${emailDomain}`,
+      `https://${emailDomain}`,
+    ].join(", ");
+  }
+
   private async getAccountIdBasedOnEmailDomainOfContacts(email: string) {
     const conn = await this.conn;
     const emailDomain = email.split("@")[1];
     const log = logger.getSubLogger({ prefix: [`[getAccountIdBasedOnEmailDomainOfContacts]:${email}`] });
     log.info("getAccountIdBasedOnEmailDomainOfContacts", safeStringify({ email, emailDomain }));
     // First check if an account has the same website as the email domain of the attendee
     const accountQuery = await conn.query(
-      `SELECT Id, Website FROM Account WHERE Website IN ('${emailDomain}', 'www.${emailDomain}', 
-                      'http://www.${emailDomain}', 'http://${emailDomain}',
-                      'https://www.${emailDomain}', 'https://${emailDomain}') LIMIT 1`
+      `SELECT Id, Website FROM Account WHERE Website IN (${this.getAllPossibleAccountWebsiteFromEmailDomain(
+        emailDomain
+      )}) LIMIT 1`
     );
     if (accountQuery.records.length > 0) {
-      const account = accountQuery.records[0] as { Id: string };
-      log.info("Found account based on email domain", safeStringify({ accountId: account.Id }));
+      const account = accountQuery.records[0] as { Id: string; Website: string };
+      log.info(
+        "Found account based on email domain",
+        safeStringify({ emailDomain, accountWebsite: account.Website, accountId: account.Id })
```

Comment:
Logging Website and accountId both it can confirm that when we matched what Website value was there for the account